docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -428,6 +428,15 @@ The encode one or more options. For example, Debug is encoded as
 ``X-B3-Flags: 1``. See more on zipkin tracing
 `here <https://github.com/openzipkin/b3-propagation>`.
 
+.. _config_http_conn_man_headers_b3:
```

Comment:
This page https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/tracing#trace-context-propagation I think should be updated to talk about a single B3 header for Zipkin.